### PR TITLE
Add brimcap launch command

### LINKED ISSUE #7: brimcap launch command
Add brimcap launch command which will take as an arg (1) the name of the pcapfile to load and (2) the 5 tuple used to describe connection that the user is searching for. brimcap launch will look in the brimcap root to find the pcapfile.

### PR DESCRIPTION
**Pull request by mattnibs**
Add the brimcap launch command. brimcap launch takes as argument the time span,
protocol, source ip and port, and destination ip and port describing
a connection and searches through all the indexed pcaps in the BRIMCAP_ROOT
packets matching the description. If the search returns a result the packets
are opened in wireshark.

Additional changes:
- add brimcap search: the same thing as brimcap launch except it writes the
  search result as a pcap file.
- brimcap index -root: If the -root flag is provided brimcap index will create
  a index of the pcap in the BRIMCAP_ROOT.

Closes #7

### CODE REVIEW

File: cmd/brimcap/launch/command.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+func (c *Command) Exec(args []string) (err error) {
+	if err := c.Command.Init(&c.rootflags, &c.searchflags, c); err != nil {
+		return err
+	}
+	defer c.Cleanup()
+
+	f, err := os.CreateTemp("", "brimcap-launch-")
+	if err != nil {
+		return err
+	}
```
Comment (nwt) [inline cmd/brimcap/launch/command.go:54]: `defer os.Remove(f.Name())` here?

File: cmd/brimcap/launch/command.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+	err = c.rootflags.Root.Search(context.TODO(), c.searchflags.Search, f)
```
Comment (nwt) [inline cmd/brimcap/launch/command.go:56]:
```suggestion
	ctx, cancel := signal.NotifyContext(context.Baackground(), os.Interrupt)
	defer cancel()
	err = c.rootflags.Root.Search(ctx, c.searchflags.Search, f)
```

File: root.go [hunk 3]
```diff
@@ -1,15 +1,32 @@
...
+	"io"
+	"io/fs"
+	"net"
 	"os"
 	"path/filepath"
 
 	"github.com/brimdata/brimcap/pcap"
-	"github.com/brimdata/zed/pkg/fs"
+	"github.com/brimdata/brimcap/pcap/pcapio"
+	fse "github.com/brimdata/zed/pkg/fs"
```
Comment (nwt) [inline root.go:15]: Nit: As a reader, I find renamed packages to be easier to follow if renamed mechanically by concatenating trailing path components, like `pks/fs` here.

File: cmd/brimcap/search/command.go [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+	out := os.Stdout
+	if c.outfile != "-" {
+		out, err = os.Create(c.outfile)
+		if err != nil {
+			return err
+		}
+	}
+
+	err = c.rootflags.Root.Search(context.TODO(), c.searchflags.Search, out)
```
Comment (nwt) [inline cmd/brimcap/search/command.go:55]:
```suggestion
	ctx, cancel := signal.NotifyContext(context.Baackground(), os.Interrupt)
	defer cancel()
	err = c.rootflags.Root.Search(ctx, c.searchflags.Search, out)
```

File: root.go [hunk 5]
```diff
@@ -26,18 +43,104 @@ func (r Root) AddPcap(path string, limit int, warner zbuf.Warner) (nano.Span, er
...
+		return fmt.Errorf("unsupported proto type: %s", req.Proto)
 	}
 
-	return index.Span(), nil
+	files, err := r.Pcaps()
+	if err != nil {
+		return err
+	}
+
+	group, _ := errgroup.WithContext(ctx)
```
Comment (nwt) [inline root.go:92]:
```suggestion
	group, cancel := errgroup.WithContext(ctx)
	defer concel()
```

**Review by nwt — APPROVED** [review]

File: cmd/brimcap/launch/command.go (hunk 1 above)
Comment (mattnibs) [reply cmd/brimcap/launch/command.go:54]: I think this needs to stay around for wireshark to read it, no?

File: root.go (hunk 5 above)
Comment (mattnibs) [reply root.go:92]: This returns a context.Context not a context.CancelFunc

**Pull request merged**